openlibrary/templates/home/scroll-to-top.html

```diff
@@ -0,0 +1,44 @@
+$def with (test)
+<style>
+#myBtn {
+  display: none;
+  position: fixed;
+  bottom: 20px;
+  right: 30px;
+  z-index: 99;
+  font-size: 18px;
+  border: none;
+  outline: none;
+  background-color:#3b98fc;
+  color: white;
+  cursor: pointer;
+  padding: 15px;
+  border-radius: 4px;
+}
+
+#myBtn:hover {
+  background-color: #555;
+}
+</style>
+<button onclick="topFunction()"id="myBtn" title="Go to top">Top</button>
+<script>
```

Comment:
Embedded Javascript is strongly discouraged. Perhaps @jdlrobson or @cdrini can suggest a good home.

Reply:
Thanks @tfmorris, I have changed the variable name. And I tried putting the files in the right directories but it did not work, I have mentioned the same in my proposed pull request.